streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/kstream/KTableTest.scala

```diff
@@ -39,29 +39,27 @@ class KTableTest extends FlatSpec with Matchers with TestDriver {
     val sinkTopic = "sink"
 
     val table = builder.stream[String, String](sourceTopic).groupBy((key, _) => key).count()
-    table.filter((_, value) => value > 1).toStream.to(sinkTopic)
+    table.filter((key, value) => key.equals("a") && value == 1).toStream.to(sinkTopic)
```

Comment:
Why do we need to change the filter condition?

Reply:
So that we can test an existing row being removed.